app/models/UserAnswers.scala

```diff
@@ -55,6 +55,30 @@ case class UserAnswers(
     }
   }
 
+    /**
+    * Sets the answer (value) to the given question (settable), and returns a failed future  if something goes 
+    * wrong - unlike set() which returns a failed Try
+    * @param settable - question we're setting the answer / data for
+    * @param value - value to set
+    * @param cleanup - true to call the question's cleanup method
+    * @tparam A - type of 'value'
+    * @return Future of updated UserAnswers
```

Comment:
Set returns a Try[UserAnswers], we then call a .get, do we want this to be a future? .get will throw exception